bindings/python/tests/test_read.py

```diff
@@ -158,6 +158,20 @@ async def test_async_reader(service_name, operator, async_operator):
         assert read_content is not None
         assert read_content == content
 
+    range_start = randint(0, len(content) - 1)
+    range_end = randint(range_start, len(content) - 1)
+
+    async with await async_operator.open(
+        filename, "rb", range_start=range_start, range_end=range_end
```

Comment:
"range_start" and "range_end" don't seem ideal to me, since Python users may not be familiar with the term "range" in this context. How about using "offset" and "size" instead?

Reply:
done, changed to offset and size